sdk/tables/azure-data-tables/inc/azure/data/tables/internal/constants.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+#pragma once
+namespace Azure { namespace Data { namespace Tables { namespace _detail {
```

Comment:
After this PR is in https://github.com/Azure/azure-sdk-for-cpp/pull/6136, we don't need this header file in the public `\inc` folder at all.

All these are implementation details for `src`, only used by `.cpp` files, so we can move this header into the `private` folder under `\src`